l10n_it_fatturapa/views/related_document_type_views.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2020 Simone Rubino - Agile Business Group
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+  -->
+
+<odoo>
+    <record id="fatturapa_related_document_type_view_tree" model="ir.ui.view">
+        <field name="name">Tree view for fatturapa.related_document_type</field>
+        <field name="model">fatturapa.related_document_type</field>
+        <field name="arch" type="xml">
+            <tree editable="bottom">
+                <field name="type"/>
+                <field name="name"/>
+                <field name="lineRef" readonly="1"/>
+                <field name="date"/>
+                <field name="numitem"/>
+                <field name="code"/>
+                <field name="cig"/>
+                <field name="cup"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="fatturapa_related_document_type_view_form" model="ir.ui.view">
+        <field name="name">Form view for fatturapa.related_document_type</field>
+        <field name="model">fatturapa.related_document_type</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="type"/>
+                            <field name="name"/>
+                            <field name="lineRef" readonly="1"/>
+                            <field name="date"/>
+                            <field name="numitem"/>
+                            <field name="code"/>
+                            <field name="cig"/>
+                            <field name="cup"/>
+                        </group>
+                        <group groups="base.group_no_one">
+                            <field name="invoice_id"/>
+                            <field name="invoice_line_id"/>
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <act_window
+        id="fatturapa_related_document_type_action"
+        name="Related documents"
+        res_model="fatturapa.related_document_type"/>
+
+    <menuitem
+        id="fatturapa_related_document_type_menu"
+        parent="fatturapa_common_menu"
+        action="fatturapa_related_document_type_action"/>
```

Comment:
Da qui quindi i `fatturapa.related_document_type` si possono creare? Perchè permettere di crearli se poi non possono essere collegati ad alcun documento?

Reply:
Ho inserito il menu solo per poterli visualizzare e avere una visione d'insieme di quali sono presenti a sistema, modifico per impedire la creazione

Reply:
:+1:

Reply:
Però sono modificabili:

![image](https://user-images.githubusercontent.com/1033131/83262478-5975ba80-a1bd-11ea-82e1-a751993e6037.png)

Reply:
sì, dici che è da impedire la modifica?
Mi ero posto il problema, ma all'interno della fattura (e ora anche dell'ordine), i `documenti correlati` sono sempre modificabili quindi li ho lasciati modificabili anche nel nuovo menu.

Se ad esempio nella fattura non fossero più modificabili dopo l'emissione della fattura, avrebbe senso renderli in sola lettura anche nel nuovo menu (idealmente, sarebbero in sola lettura quelli collegati a fatture emesse).

Reply:
Ah, ok. In generale in effetti non penso abbia senso modificarli dopo aver inviato la fattura elettronica.
Quindi qui li metterei in sola lettura. Poi sarebbero da mettere in sola lettura anche in fattura.

Reply:
Li ho messi in sola lettura nel nuovo menu, puoi aggiornare le review?
Modificare la vista della fattura è un miglioramento che non dipende da questa PR quindi penso sia da fare in un'altra issue/PRs dedicate.

Reply:
> Modificare la vista della fattura è un miglioramento che non dipende da questa PR quindi penso sia da fare in un'altra issue/PRs dedicate

Non l'hai già modificata in https://github.com/OCA/l10n-italy/pull/1708/commits/30c514689f9ed23b246e41e0f4d1c9d307354df6 ?

Reply:
Ho estratto la vista `tree` che era scolpita nella fattura in modo da poterla riutilizzare senza dover ripetere il codice, ma il comportamento è rimasto invariato.

Impostare la vista in 'sola lettura' è una modifica al comportamento che, come dicevo,
> non dipende da questa PR quindi penso sia da fare in un'altra issue/PRs dedicate.